src/components/caption/index.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "@testing-library/react";
+
+import Caption from ".";
+
+describe("Caption", () => {
+	it("should render title, credit and caption", () => {
+		render(<Caption title="title" credit="credit" caption="caption" />);
+
+		expect(screen.queryByText("title")).not.toBeNull();
+		expect(screen.queryByText("credit")).not.toBeNull();
+		expect(screen.queryByText("caption")).not.toBeNull();
+	});
+	it("should render string child with additional classes", () => {
+		const { container } = render(<Caption title="Hello World" className="test-class" />);
+		// not quite best practice with rtl's methodology of not testing for classes
+		// couldn't find a way to test for className without using its role or text
```

Comment:
testing-library (basis of RTL) has `.toHaveClass('')` ... I think people get hung up on the "not testing implementation" to include classes, but classes are used to define appearance which is what the library claims is the best way to test.

https://github.com/testing-library/jest-dom#tohaveclass

Reply:
gotcha gotcha -- thanks